bin/container.sh

```diff
@@ -237,9 +229,10 @@ console_mysql() {
 }
 
 status() {
-  eval $(docker-machine env ${project}) 2>/dev/null
-  docker-machine active 2>/dev/null | grep ${project} || usage "start ${project}" 1
-  eval $(docker-machine env ${project})
+  if [ "$OS" == "OSX" ]; then
+    docker-machine active 2>/dev/null | grep ${project} || usage "start ${project}" 1
```

Comment:
ok ... we can add the eval back in.

there is an always-invoked eval earlier on in the call stack and as such i had thought this was redundant. but if it is needed, we can add it back in.